system/forms/preside-objects/asset/group.upload.xml

```diff
@@ -11,6 +11,7 @@ that have been uploaded.
 		<fieldset id="standard" sortorder="10">
 			<field sortorder="10" binding="asset.asset_folder" control="assetFolderPicker" quickAdd="true" />
 			<field sortorder="20" binding="asset.author" control="textinput" />
+			<field sortorder="30" binding="asset.alt_text" control="textinput" />
```

Comment:
I'm not sure this makes sense here as this will set the same alt text for all images uploaded in a batch upload process. It should either just not be here, or added to the mini form you get for each asset (currently just allows you to edit the label).